Frameworks/AEPSwiftUI/Sources/ContentCards/Template/SmallImageTemplate.swift

```diff
@@ -0,0 +1,90 @@
+/*
+ Copyright 2024 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import AEPMessaging
+import SwiftUI
+
+/// A class representing a content card template with a small image layout.
+///
+/// `SmallImageTemplate` is a subclass of `BaseTemplate` and conforms to the `ContentCardTemplate` protocol.
+/// It provides a structured layout for content cards that include a small image, title, body, and optional buttons.
+/// This class is initialized with `ContentCardSchemaData`, which is used to populate the template's properties.
+/// If the required `title` is missing from the schema data, the initialization fails.
+///
+/// - Note: The `view` property is lazily initialized and represents the entire layout of the content card.
+public class SmallImageTemplate: BaseTemplate, ContentCardTemplate {
+    var templateType: ContentCardTemplateType = .smallImage
```

Comment:
should this be a `let`?

Reply:
This var is an implementation of the protocol ContentCardTemplate

```
protocol ContentCardTemplate: BaseTemplate {
    var templateType: ContentCardTemplateType { get }
    /// other implementation
}
```

Since it is a protocol property.. I cannot keep this as a let constant.